### DISCUSSION TITLE
ES modules Import @react-email/components error

### DISCUSSION THREAD

**Discussion by ninnjak** [resend/react-email#703] [Q&A] [answered]
I have an ES module node application that imports `@react-email/components`. The issue is it pulls in the CommonJs version instead of the MJS version which throws the following error:

```
SyntaxError: Named export 'Body' not found. The requested module '@react-email/components' is a CommonJS module, which may not support all module.exports as named exports.
```

Repro: https://codesandbox.io/p/sandbox/adoring-ride-lk3zh7

If I add the following to `@react-email/components` package.json it fixes my issue:
```
"exports": {
  ".": {
    "import": "./dist/index.mjs",
    "require": "./dist/index.js",
    "types": "./dist/index.d.ts"
  }
},
```

Would this be something that needs to be added to `@react-email/components` or is there something i'm doing wrong on my end?

**Comment by bukinoshita** [comment, marked as answer]
We just added here https://github.com/resendlabs/react-email/pull/709
